sync2jira/main.py

```diff
@@ -415,7 +415,7 @@ def main(runtime_test=False, runtime_config=None):
     config['validate_signatures'] = False
 
     try:
-        if INITIALIZE:
+        if bool(INITIALIZE):
```

Comment:
Ignore this, this will be overwritten by this PR: https://github.com/sidpremkumar/Sync2Jira/pull/84/files#diff-5d7201dc63e5ab6f932402c3c6a6eb9bR418